gprofiler/profilers/perf.py

```diff
@@ -237,6 +237,18 @@ class PerfMetadata(ApplicationMetadata):
     def relevant_for_process(self, process: Process) -> bool:
         return False
 
+    def add_exe_metadata(self, process: Process, metadata: Dict[str, Any]) -> None:
+        try:
+            static = is_statically_linked(f"/proc/{process.pid}/exe")
+        except FileNotFoundError:
+            raise NoSuchProcess(process.pid)
+
+        exe_metadata = {"link": "static" if static else "dynamic"}
+        if not static:
+            exe_metadata["libc"] = "musl" if is_musl(process) else "glibc"
```

Comment:
The `metadata["libc"] = None` part was on purpose - so that we know the profiler decided not to collect the field (and not missing for whatever reason)